spec/system/default_search_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+require 'spec_helper'
+
+describe "Default search", type: :system do
```

Comment:
I am not sure this should be a "system" test or if it matters at all that is tagged as such, but tagging it as "system" is how I was able to prevent Rubocop from complaining that: "RSpec/DescribeClass: The first argument to describe should be the class or module being tested"

Notice that this test is issuing a search against Solr directly, not against the entire webpage page workflow (hence I am not sure if it should be "system"). But would like to keep the test narrow in scope and test the indexing pipeline alone rather than the entire web page display workflow.